tools/rimage/src/module.c

```diff
@@ -415,14 +419,33 @@ void module_parse_sections(struct module *module, const struct memory_config *me
 	sections_info_finalize(&module->bss);
 
 	size_t fw_size = module->data.size + module->text.size;
+	size_t rounded, start, end, size;
 
 	fprintf(stdout, " module: input size %zd (0x%zx) bytes %d sections\n",
 		fw_size, fw_size, module->num_sections);
-	fprintf(stdout, " module: text %zu (0x%zx) bytes\n"
-		"\tdata %zu (0x%zx) bytes\n"
-		"\tbss  %zu (0x%zx) bytes\n\n",
-		module->text.size, module->text.size,
-		module->data.size, module->data.size,
+
+	/* MST_TEXT */
+	start = module->text.start;
+	end = module->text.end + cold_text_size;
+	size = module->text.size + cold_text_size;
+	rounded = ALIGN_UP(end, MAN_PAGE_SIZE) - start;
+	fprintf(stdout,
+		" module: text %zu (0x%zx) bytes, including %zu (0x%zx) %zu%% efficiency bytes\n",
```

Comment:
"efficiency bytes" ?

Reply:
efficient memory i.e. not higher power SRAM